lib/generators/rails/presenter_generator.rb

```diff
@@ -0,0 +1,12 @@
+module Rails
+  class PresenterGenerator < Rails::Generators::NamedBase
+    source_root File.expand_path("../templates", __FILE__)
```

Comment:
Use expand_path('templates', __dir__) instead of expand_path('../templates', __FILE__).